src/containers/DeveloperPortal/Tools/Tools.scss

```diff
@@ -0,0 +1,70 @@
+@import 'styles/font-mixins';
+
+.Tools {
+  background: url('./background-pattern.png');
+  background-color: #f6f9fc;
+  background-position: center;
+  background-size: cover;
+
+  &__container {
+    padding: 24px 24px;
+
+    @media (min-width: 768px) {
+      padding: 48px 48px;
+    }
+
+    @media (min-width: 1366px) {
```

Comment:
this is a nitpick, but can we change these to `max-width` to preserve the consistency in our codebase? haha

Reply:
oh but i thought the designs were based on `min-width`s? as told by MR K

Reply:
yeah the designs are like that, but you can translate that to max-width?

Reply:
ok give me a couple of mins

Reply:
take your time! i gotta go spend time with the family tonight. but everything looks good, i'll merge it in tomorrow